nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -81,6 +81,11 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         # Setup the tokenizer
         self._setup_tokenizer(cfg.tokenizer)
 
+        # Assert config has "prompt_format"
+        if "prompt_format" not in cfg:
```

Comment:
should we add it to config with `???` and in the documentation of `speech_to_text_aed.py` script?

Reply:
I did add it to config with Canary as default I think

Reply:
yes, canary is default in yaml config.